app/experimenter/experiments/views.py

```diff
@@ -72,6 +73,33 @@ def get_owner_display_value(self):
         if user_id is not None:
             return str(get_user_model().objects.get(id=user_id))
 
+    def get_display_start_date_info(self):
```

Comment:
It didn't work to move this method to the filterset. I put it here in the form because that's where other methods called in the template are located... what am I missing?

Reply:
Okay, keep it here, see if you can add

- experiment_date_field
- date_range_before
- date_range_after

to Meta.fields in this form and see if that works or if things blow up

Reply:
I made a general PR comment that addressed my questions with this!